typescript/sdk/src/gas/HyperlaneIgpChecker.ts

```diff
@@ -72,8 +72,8 @@ export class HyperlaneIgpChecker extends HyperlaneAppChecker<
 
     await this.checkBytecode(
       chain,
-      'OverheadIGP',
-      contracts.defaultIsmInterchainGasPaymaster.address,
+      'InterchainGasPaymaster',
+      contracts.interchainGasPaymaster.address,
       [BytecodeHash.OVERHEAD_IGP_BYTECODE_HASH],
```

Comment:
we probably need to change this? cc @nambrot

Reply:
cc @nambrot

Reply:
refer to https://github.com/hyperlane-xyz/issues/issues/642